sandboxed_containers/sandboxed-containers-release-notes.adoc

```diff
@@ -0,0 +1,167 @@
+
+[id="sandboxed-containers-1-2-new-features-and-enhancements"]
+== New features and enhancements
+
+[id="kata-metrics-rn"]
+=== Kata specific metrics and dashboard
+
+The {sandboxed-containers-operator} now deploys an osc-monitor daemonset. This enables the collection of metrics specific to workloads running in sandboxed containers (including metrics about the hypervisor and the Guest OS instances). In addition, a pre-configured dashboard provides insights into {sandboxed-containers-first} components, such as the total number of lightweight VMs enabled in a cluster and the CPU/memory consumption per VM.
+ All metrics, as well as the dashboard, are available in the web console. For more information, see xref:../sandboxed_containers/monitoring-sandboxed-containers.adoc#monitoring-sandboxed-containers[Monitoring {sandboxed-containers-first}].
+
+[id="enhanced-logging-rn"]
+=== Enhanced logging
+
+Administrators can now collect enhanced logs for {sandboxed-containers-first} runtime components. Enhanced logs are available when the CRI-O log level is set to `debug`. These logs are collected by the must-gather tool, or can be viewed in the node journal.
+For more information, see xref:../sandboxed_containers/troubleshooting-sandboxed-containers.adoc#sandboxed-containers-enable-debug-logs_troubleshooting-sandboxed-containers[Enabling debug logs for {sandboxed-containers-first}].
+
+[id="node-eligibility-rn"]
+=== Check node eligibility to run {sandboxed-containers-first}
+
+Administrators can now check the eligibility of cluster nodes to run {sandboxed-containers-first}. This feature uses the Node Feature Discovery (NFD) operator to detect  node capabilities. Eligible nodes are labeled with `feature.node.kubernetes.io/runtime.kata`, and the {sandboxed-containers-operator} uses this label to select candidate nodes for installation.
+
+The administrator must deploy the NFD operator to use this feature, create a specific *NodeFeatureDiscovery* custom resource, and enable `checkNodeEligibility' when creating the *KataConfig* custom resource.
+//For more information, see xref:../sandboxed_containers/deploying-sandboxed-container-workloads.adoc#sandboxed-containers-check-nodes_deploying-sandboxed-containers[Checking the eligibility of cluster nodes to run {sandboxed-containers-first}].
+
+[id="OSC-compatibility-with-CNV"]
+=== {sandboxed-containers-first} compatibility with {VirtProductName}
+
+Users can now run {sandboxed-containers-first} on clusters with {VirtProductName} when VMs are configured correctly.
+For more information, see xref:../sandboxed_containers/understanding-sandboxed-containers.adoc#sandboxed-containers-with-other-ocp-components_understanding-sandboxed-containers[Using {sandboxed-containers-first} with {VirtProductName}]
+
+[id="support-AWS-bare-metal"]
+=== {sandboxed-containers-first} availability on AWS bare-metal (tech preview)
```

Comment:
Only hyphenate bare metal when used as an adjective for disambiguation, for example: "bare-metal installation". Also, we refer to it as "Technology Preview" in docs.
```suggestion
=== {sandboxed-containers-first} availability on AWS bare metal (Technology Preview)
```